code/build.R

```diff
@@ -396,6 +396,26 @@ command_tippecanoe = paste("tippecanoe -o rnet_{date_folder}.pmtiles",
 
 responce = system(command_tippecanoe, intern = TRUE)
 
+# Simplified network tiling
+setwd("outputdata")
+# Check the combined_network_tile.geojson file is there:
+file.exists("simplified_network.geojson")
+command_tippecanoe = glue::glue("tippecanoe -o snet_{date_folder}.pmtiles",
```

Comment:
Can you rename this as

```suggestion
command_tippecanoe = glue::glue("tippecanoe -o rnet_simplified_{date_folder}.pmtiles",
```

?

Reply:
> Can you rename this as
> 
> ?

got it will do